### ISSUE TITLE
Repeat in EFF wordlist

### ISSUE THREAD

**Issue by J-J-Chiarella** [keepassxreboot/keepassxc#7962]
## Overview

The EFF wordlist (5-dice list) has two words that repeat <https://www.eff.org/files/2016/07/18/eff_large_wordlist.txt> at 66622 and 66631. KeePassXC also uses this flawed list <https://github.com/keepassxreboot/keepassxc/blob/develop/share/wordlists/eff_large.wordlist> at lines 7745 and 7750.

These words are "yo-yo" and "yoyo."

Not only is this the *same* word with two different spellings, but the form at 7745 (yo-yo) is the only word that has a hyphen, which is usually reserved to separate words when password systems do not even allow single spaces between printable characters, which is still frightfully common.

## Steps to Reproduce
1. Look at KeePassXC wordlist <https://github.com/keepassxreboot/keepassxc/blob/develop/share/wordlists/eff_large.wordlist>.
2. Note that the word at line 7745 (yo-yo) is the same as the word at line 7750 (yo-yo), with the only exception being the absence or presence of a hyphen. The word at line 7745 is the only word on the list with a hyphen, too. The hyphen is supposed to be reserved for separating diceware words in password entry when not even a single space between printable characters is allowed.

## Expected Behavior
7745 and 7750 should have totally different words. To keep alphabetical order, I suggest "yes" for the word at line 7745 and "young" for the word at line 7750.

## Context

The dmuth Diceware site <https://diceware.dmuth.org/> changed this entry due to a similar bug report <https://github.com/dmuth/diceware/issues/14>.

The EFF general short word list <https://www.eff.org/files/2016/09/08/eff_short_wordlist_1.txt> also has this repetition of "yo-yo" and "yo-yo" for entries 6652 and 6655, respectively.

Operating System: ALL

Desktop Env: ALL

Windowing System: ALL

**Comment by droidmonkey** [comment]
Please just submit a PR that corrects this. In the grand scheme of things this is incredibly low risk. If you happen to get the word randomly you can just manually edit it or refresh to get other words.

**Comment by J-J-Chiarella** [comment]
Thank you. I went and made the edits. I also noticed that someone in the past removed three words for possibly being offensive. I added a few to restore the order. I also, by happenstance, noticed the inclusion of both "hankie" and "hanky." Because the phrase "hanky-panky" uses that spelling and by analogy to some former slang like "cookie," I suggested keeping "hankie" and swapping out "hanky."

In two cases I needed to replace another word to make room for a replacement, but the order is preserved and there are no breaking changes between wordlists. The "problem" words no longer exist instead of being reshuffled. Nothing downstream of the changes is changed in ordering or number value.

Maybe another project will find the amended list useful.
